test/unit/controllers/TokenAPI.ws.spec.js

```diff
@@ -40,7 +40,8 @@ describe('TokenAPI WS', () => {
         deviceHive = new DeviceHive({
             login: `dhadmin`,
```

Comment:
We can move test configuration for connection in separate file to reuse in each test

Reply:
This configuration is related to some stub servers in tests, so, from my perspective it would be better to leave connection configuration in the same file as stubs configuration, until stubs would be reworked.